linuxNew/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec

```diff
@@ -0,0 +1,208 @@
+%global upstream_version 8.0.302+8
+# Only [A-Za-z0-9.] allowed in version:
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
+# also not very intuitive:
+#  $ rpmdev-vercmp 8.0.302.0.1___8 8.0.302.0.0+8
+#  8.0.302.0.0___8 == 8.0.302.0.0+8
+%global spec_version 8.0.302.0.0.8
+%global spec_release 1
+%global priority 1111
+
+%global source_url_base https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download
```

Comment:
This will need to be: `https://github.com/adoptium/temurin8-binaries/releases/download`